src/client/bots/GreedyBot.hpp

```diff
@@ -0,0 +1,23 @@
+#ifndef GreedyBot_hpp
+#define GreedyBot_hpp
+
+#include "../../common/util/InputDef.hpp"
+
+#include <ctime>
+#include <cstdlib>
+
+class GreedyBot {
```

Comment:
Worth having a base bot class that this inherits from

Reply:
yeah I had this fancy idea of strategy's and a bot which comes to life when you give it one, and even an idea of "tactics" which override strategies and come into effect in combat scenarios for example, but Im gonna just implement the different bots, see if there's shared functionality a base class could help with then refactor if it seems meaningful